c_binding/include/libstoragemgmt/libstoragemgmt_hash.h

```diff
@@ -26,25 +26,25 @@
 extern "C" {
 #endif
 
-/**
+/*
```

Comment:
The hash seems only used by plugin, no client functions require the use of `lsm_hash`. And I don't include plugin documentation for this PR yet.